octobot_evaluators/matrix/matrix_manager.py

```diff
@@ -298,7 +325,10 @@ def is_tentacle_value_valid(matrix_id, tentacle_path, timestamp=0, delta=10) ->
     if timestamp == 0:
         timestamp = time.time()
     try:
-        return timestamp - (get_tentacle_node(matrix_id, tentacle_path).node_value_time +
+        node = get_tentacle_node(matrix_id, tentacle_path)
+        if node is None:
```

Comment:
```suggestion
        if node := get_tentacle_node(matrix_id, tentacle_path) is None:
```

Reply:
Actually we can't do that is it will assign the result of `get_tentacle_node(matrix_id, tentacle_path) is None` to `node` (the bool)